source/Nevermore/Querying/AST/Select.cs

```diff
@@ -23,8 +25,12 @@ public string GenerateSql()
         {
             var orderByString = orderBy != null ? $@"
 {orderBy.GenerateSql()}" : string.Empty;
+            
+            var groupByString = groupBy != null ? $@"
+{groupBy.GenerateSql()}" : string.Empty;
```

Comment:
since `null` interpolates to empty string, could we simplify this to `var groupByString = groupBy?.GenerateSql()`

![image](https://user-images.githubusercontent.com/5088479/123029922-4582a380-d425-11eb-952d-beb0ff3ffed1.png)

Reply:
If I move the carriage return to the GenerateSql method yes.  Done